techniques/system/common/1.0/promises.st

```diff
@@ -494,7 +494,7 @@ bundle agent check_log_system
 
     !windows.rsyslogd.!policy_server::
       "/etc/rsyslog.d/rudder-agent.conf"
-        edit_line => append_if_no_lines("#Rudder log system${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then @@${server_info.cfserved}:&SYSLOGPORT&${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then ~"),
+        edit_line => append_if_no_lines("#Rudder log system${const.n}$RepeatedMsgReduction off${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then @@${server_info.cfserved}:&SYSLOGPORT&${const.n}if $syslogfacility-text == 'local6' and $programname startswith 'rudder' then ~"),
```

Comment:
This only addresses the rsyslog case. Is there no equivalent for syslog-ng? syslog?